src/libs/ui/src/atoms/ProductDetailsPageFavoriteButton/index.tsx

```diff
@@ -0,0 +1,58 @@
+// FavoriteButton.tsx
+import React, { useState } from 'react';
+import { toast, ToastContainer, ToastOptions } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import './ProductDetailsPageFavoriteButton.styles.scss';
+
+
+const FavoriteButton: React.FC = () => {
+  const [isFavorite, setIsFavorite] = useState(false);
+
+  const handleClick = () => {
+    setIsFavorite((prev) => !prev);
+
+    // Define toast options
+    const toastOptions: ToastOptions = {
+      position: "top-right",
+      autoClose: 2000, // 2 seconds
+      hideProgressBar: false,
+      closeOnClick: true,
+      pauseOnHover: true,
+      draggable: true,
+      progress: undefined,
+      
+    };
+
+    // Display toast with custom content
+    toast.success(
+      <div className='flex items-center'>
+        <div className='favorite-icon-outline favorite-icon-filled'></div>
+        <span className='ml-2'>
+          {isFavorite ? 'Removed from Favourites!' : 'Added to Favourites!'}
+        </span>
+      </div>,
+      toastOptions
+    );
+  };

```

Comment:
create toast success toast and error toast functions in utils and move these toast success for reusability, functions should accept message and position